docs/docs/apis/openidoauth/scopes.md

```diff
@@ -22,14 +22,16 @@ ZITADEL supports the usage of scopes as way of requesting information from the I
+| `urn:zitadel:iam:org:project:role:{rolename}`     | `urn:zitadel:iam:org:project:role:user`                 | By using this scope a client can request the claim urn:zitadel:iam:roles:rolename} to be asserted when possible. As an alternative approach you can enable all roles to be asserted from the [project](../../guides/manage/console/projects) a client belongs to.     |
+| `urn:zitadel:iam:org:domain:primary:{domainname}` | `urn:zitadel:iam:org:domain:primary:acme.ch`            | When requesting this scope **ZITADEL** will enforce that the user is a member of the selected organization. If the organization does not exist a failure is displayed                                                                                                 |
+| `urn:zitadel:iam:role:{rolename}`                 |                                                         |                                                                                                                                                                                                                                                                       |
+| `urn:zitadel:iam:org:project:id:{projectid}:aud`  | `urn:zitadel:iam:org:project:id:69234237810729019:aud`  | By adding this scope, the requested projectid will be added to the audience of the access and id token                                                                                                                                                                |
+| `urn:zitadel:iam:user:metadata`                   | `urn:zitadel:iam:user:metadata`                         | By adding this scope, the metadata of the user will be included in the token. The values are base64 encoded.                                                                                                                                                          |
+| `urn:zitadel:iam:user:resourceowner`              | `urn:zitadel:iam:user:resourceowner`                    | By adding this scope, the resourceowner (id, name, primary_domain) of the user will be included in the token.                                                                                                                                                         |
+| `urn:zitadel:iam:org:idp:id:{idp_id}`             | `urn:zitadel:iam:org:idp:id:76625965177954913`          | By adding this scope the user will directly be redirected to the identity provider to authenticate. Make sure you also send the primary domain scope if a custom login policy is configured. Otherwise the system will not be able to identify the identity provider. |
+
+:::note
+If access to ZITADEL's API's is needed, e.g. with a service user the scope `urn:zitadel:iam:org:project:id:ZITADEL:aud` needs to be used with the authorization request
+:::
```

Comment:
Add this as scope to the table

Reply:
I would also prefer lowercase writing of this

Reply:
first, i've implemented it lowercase and then noticed that it's uppercase in the issue, so i changed it 🙃

Reply:
IMO the uppercase `ZITADEL` is only the marketing and documentation usage of the name.

Other then that i think we should use the lowercase `zitadel` for all other things 😁